src/clib/pio_darray.c

```diff
@@ -11,6 +11,10 @@
 #include <config.h>
 #include <pio.h>
 #include <pio_internal.h>
+#ifdef PIO_MICRO_TIMING
```

Comment:
Was the GPTL timer that's already part of PIO not working for you?

Or do you believe we should remove GPTL?